tests/test.py

```diff
@@ -33,28 +47,173 @@ def test_multi_line_condition():
 # GOOD (use parentheses for line continuation)
 def test_parentheses_in_if_on_new_line():
     s = """if (
-        a == b
-        ): c + d
+a == b
+): c + d
     """
     assert not _results(s)
 
 
 # BAD (use parentheses in both case of line continuation)
 def test_parentheses_in_if_only_with_second_new_line():
-    s = """if (a == b
-        ): c + d
+    s = """if ( a == b
+): c + d
     """
-    assert _results(s)
+    assert not _results(s)
 
 
 # BAD (use parentheses in both case of line continuation)
 def test_parentheses_in_if_only_with_first_new_line():
     s = """if (
-    a == b): c + d
+a == b): c + d
     """
     assert _results(s)
 
 
+# GOOD (have all brackets on the same line)
+def test_list_in_one_line():
+    s = """a = [1, 2, 3]"""
+    assert not _results(s)
+
+
+# GOOD
+# (when the opening bracket is the last thing on a line, the matching closing
+# bracket is the first thing on a new line that matches the indentation of the
+# line with the opening bracket)
+def test_list_with_enters_line():
+    s = """a = [
+1, 2, 3
+]"""
+    assert not _results(s)
+
+
+# BAD (opening bracket is last, but closing is not on new line)
+def test_list_with_only_one_enter_line():
+    s = """a = [
+1, 2, 3]"""
+    assert _results(s)
+
+
+# BAD
+# (opening bracket is last, closing is on new line but indentation mismatch)
+def test_list_mismatch_line():
+    s = """a = [
+1, 2, 3
+    ]"""
+    assert _results(s)
+
+
+# GOOD (opening bracket is not last, so we don't care about the closing one)
+def test_list_open_bracket_not_last():
+    s = """a = [1, 2, 3
+]"""
+    assert not _results(s)
+
+
+# GOOD (opening bracket is not last, so we don't care about the closing one)
+def test_list_open_bracket_not_last_2():
+    s = """a = [1, 2, 3
+]"""
+    assert not _results(s)
+
+
+# (pretty much the same rules apply for multiple brackets)
+
+# GOOD (have all brackets on the same line)
+def test_multilist():
```

Comment:
Usually, it's called nested or multi dimensional list/array/...
```suggestion
def test_nested_list():
```
also applies to other test names